README.md

```diff
@@ -0,0 +1,116 @@
+## Rio-tiler
+
+Rasterio plugin to serve mercator tiles from satellite imagery hosted on Amazon S3 buckets.
+
+![circleci][circleci_img]
+![codecov][codecov_img]
+
+[circleci_img]: https://circleci.com/gh/mapbox/rio-tiler.svg?style=svg&circle-token=b78bc1a238c21046a855a9c80b441a8f2f9a4478
+[codecov_img]: https://codecov.io/gh/mapbox/rio-tiler/branch/master/graph/badge.svg?token=zuHupC20cG
+
+Additional support is provided for the following satellite missions: 
+
+* Sentinel 2
+* Landsat 8
+* CBERS
+
+Rio-tiler supports Python 2.7 and 3.3-3.6.
+
+### Install
+
+You can install rio-tiler using pip
+
+```
+    $ pip install -U pip
+    $ pip install rio-tiler
+```
+
+or from source:
+
+```
+
+    $ git clone https://github.com/mapbox/rio-tiler.git
+    $ cd rio-tiler
+    $ pip install -U pip
+    $ pip install -e .
+
+```
+
+here is how to create an AWS Lambda package on most UNIX machines:
+
+```
+    $ pip install rio-tiler --no-binary numpy -t /tmp/vendored -U
+    $ zip -r9q package.zip vendored/*
+```
+
+### Overview
+
+Create tiles using one of these rio_tiler modules: `main`, `sentinel2`, `landsat8`, `cbers`. 
+
+The `main` module can create mercator tiles and get the bounds of any satellite imagery in an AWS S3 bucket that you have read access to. The other, mission specific, modules can also return metadata. 
```

Comment:
rio-tiler.main is not limited to AWS files but support any files supported by rasterio (local, http ... )

Reply:
woah! cool! I hadn't realized that